src/platforms/stm32/src/lib/sys.c

```diff
@@ -26,6 +26,8 @@
 
 #include "gpiodriver.h"
 
+#include <trace.h>
```

Comment:
We should align <> and "" usage.

Reply:
agreed, no further actions on this PR about this, we'll handle this with future cleanups.